### PR TITLE
fix/issue_27

### PR DESCRIPTION
**Pull request by AymenFJA**
fixing

1. The name of the branch is changed from `fix_ransac` to `fix/issue_27`  to match the naming policy for any branch that fixing an issue in the code .

2. Fixing   `H, mask = cv2.findHomography(keys1_ndarray, keys2_ndarray, method=cv2.RANSAC, ransacReprojThreshold=ransac_threshold, maxIters=2000)
TypeError: 'maxIters' is an invalid keyword argument for this function `

  by using the default `H, mask = 
  cv2.findHomography(keys1_ndarray,keys2_ndarray,cv2.RANSAC,ransac_threshold,2000)`

### CODE REVIEW

File: src/phase_3_ransac_filtering/out.csv [hunk 1]
```diff
@@ -0,0 +1,7086 @@
+x1,y1,sigma1,x2,y2,sigma2
```
Comment (iparask) [inline src/phase_3_ransac_filtering/out.csv]: Is this file needed?

**Review by iparask — CHANGES_REQUESTED** [review]
Hello @AymenFJA, can you remove the `*.pyc` files from this fix?

**Review by iparask — CHANGES_REQUESTED** [review]
There are still `*.pyc` files. Can you remove them?

File: src/phase_3_ransac_filtering/out.csv (hunk 1 above)
Comment (AymenFJA) [reply src/phase_3_ransac_filtering/out.csv]: out.csv is the filtered key points. I did remove it.

**Review by iparask — APPROVED** [review]

**Comment by iparask** [comment]
We will move forward with this PR based on @mmacferrin comments. From my perspective it is okay.

**Comment by mmacferrin** [comment]
Thanks @AymenFJA @iparask.
